disclosure_backend/settings.py

```diff
@@ -1,7 +1,8 @@
 import os
+import os.path as op
```

Comment:
FWIW, I prefer the verbosity here. I don't think it's crazy to write out the `os.path` namespace. Alternatively, I'd prefer `path` over `op` because it's more descriptive.